utils/constants.js

```diff
@@ -5,4 +5,5 @@ export const ACCOUNT_STATUS = {
 export const POST_STATUS = {
   UNPUBLISH: 'UNPUBLISH',
   PUBLISH: 'PUBLISH',
+  NOTIFICATION: 'Notification',
 }
```

Comment:
This file seems no longer used, you should check its reference (switch to /shared/constants.js if have) and delete it.

Reply:
this file of Phuc Nguyen create. I customed it